src/vcpkg/base/unicode.cpp

```diff
@@ -190,15 +191,15 @@ namespace vcpkg::Unicode
         {
             switch (static_cast<utf8_errc>(condition))
             {
-                case utf8_errc::NoError: return "no error";
-                case utf8_errc::InvalidCodeUnit: return "invalid code unit";
-                case utf8_errc::InvalidCodePoint: return "invalid code point (>0x10FFFF)";
-                case utf8_errc::PairedSurrogates:
-                    return "trailing surrogate following leading surrogate (paired surrogates are invalid)";
-                case utf8_errc::UnexpectedContinue: return "found continue code unit in start position";
-                case utf8_errc::UnexpectedStart: return "found start code unit in continue position";
-                case utf8_errc::UnexpectedEof: return "found end of string in middle of code point";
-                default: return "error code out of range";
+                case utf8_errc::NoError: return msg::format(msgNoError).extract_data();
+                case utf8_errc::InvalidCodeUnit: return msg::format(msgInvalidCodeUnit).extract_data();
+                case utf8_errc::InvalidCodePoint:
+                    return msg::format(msgInvalidCodePoint).append_raw(" (>0x10FFFF)").extract_data();
```

Comment:
I don't see any reason to exclude the (>0x10FFFF) bit from the localized message?

Reply:
The message is reused with a different code point

Reply:
Makes sense!